index.html

```diff
@@ -4,15 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
-    </div>
+    <nav class="Navbar">
+        <div class="top-logo">
+            <h2>My Portfolio</h2>
```

Comment:
- [ ] Please use `<a>` tag for the logo and put a link to your home page, so upon clicking logo, the user is navigated to the `Home` page. This is for three reasons: convention, semantics, and user experience(UX). It also has the added benefit of being more accessible (especially if you use an alt attribute on your logo), which makes search engine bots happier. That said, I would not use an `<h1>` tag for a logo since the heading tags should structure your HTML.